Run Test on Node v12 Failed

There are multiple opened PRs are failing due to the following errors: 

 83) GraphQL-HTTP tests for restify
       Custom result extensions
         does nothing if extensions function does not return an object:
     DeprecationWarning: The legacy HTTP parser is deprecated.
      at <anonymous>
      at internalBinding (internal/bootstrap/loaders.js:136:34)
      at process.binding (internal/bootstrap/loaders.js:113:14)
      at Object.<anonymous> (node_modules/http-deceiver/lib/deceiver.js:22:24)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at Object.<anonymous> (node_modules/spdy/lib/spdy/handle.js:5:20)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at Object.<anonymous> (node_modules/spdy/lib/spdy.js:6:15)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at Object.<anonymous> (node_modules/restify/lib/server.js:14:12)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at Object.<anonymous> (node_modules/restify/lib/index.js:10:14)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at /home/runner/work/express-graphql/express-graphql/src/__tests__/http-test.ts:133:21
      at Context.<anonymous> (src/__tests__/http-test.ts:2338:19)
      at processImmediate (internal/timers.js:461:21)



npm ERR! code ELIFECYCLE
npm ERR! errno 83
npm ERR! express-graphql@0.12.0 testonly: `mocha src/**/__tests__/**/*.ts`
npm ERR! Exit status 83
npm ERR! 
npm ERR! Failed at the express-graphql@0.12.0 testonly script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-04-13T02_03_31_597Z-debug.log
Error: Process completed with exit code 83.

--- a/.mocharc.yml
+++ b/.mocharc.yml
@@ -1,4 +1,3 @@
-throw-deprecation: true
 check-leaks: true
 require:
   - './resources/register.js'

